src/engine/renderer/tr_shader.cpp

```diff
@@ -5197,16 +5208,17 @@ static void CollapseStages()
 
 		// Finally disable useless heightMapInNormalMap if both normal and relief mapping are disabled.
 		// see https://github.com/DaemonEngine/Daemon/issues/376
-		stage->isHeightMapInNormalMap = stage->isHeightMapInNormalMap && ( stage->enableNormalMapping || stage->enableReliefMapping );
+		stage->hasHeightMapInNormalMap &= ( stage->enableNormalMapping || stage->enableReliefMapping );
```

Comment:
same thing about writing out the full condition

Reply:
Done full condition again.